### ISSUE TITLE
Sending scroll wheel message to the TCP does not really work...

### ISSUE THREAD

**Issue by martinfabian** [martinfabian/MFXlist#2]
Fix the scrolling, Mousewheel scroll over MFXlist sends message to TCP, but the arrange view is scrolled! 
Also, Ctrl+Mousewheel in track area does not work correctly.

Is this a focus issue?

**Comment by martinfabian** [comment]
Some of the scroll wheel issues have been fixed.
- Plain Wheel now scrolls (almost) as expected
- Shit+Wheel behaves as expected
- Ctrl+Wheel does not really work as expected, see above
- Alt+Wheel seems to work as expected
- Shift+Ctrl+Wheel seems to work as expected
- Shift+Alt+Wheel is not assigned anything here, so... I guess it works as expected
- Ctrl+Alt+Wheel works as expected
- Shift+Ctrl+Alt+Wheel is not assigned anything here, so...

Basically, only Ctrl+Wheel does not work as expected in v 0.9.0.
